prive/run.py

```diff
@@ -0,0 +1,97 @@
+"""
```

Comment:
Should the name of the executable should be `prive` ? 

(I don't know how to run Python scripts -- do you need a shell-script wrapper?)

Reply:
I originally had it as that and the tests failed, I think because the package wasn't being installed correctly anymore (perhaps it was just installing the single `prive.py` file?)